test/astar.cc

```diff
@@ -109,8 +97,17 @@ std::pair<vb::GraphId, vm::PointLL> f({tile_id.tileid(), tile_id.level(), 5}, {0
 std::pair<vb::GraphId, vm::PointLL> g({tile_id.tileid(), tile_id.level(), 6}, {0.05, 0.11});
 } // namespace node
 
-#ifdef MAKE_TEST_TILES
+bool file_exists(const std::string& fname) {
```

Comment:
we have `filesystem::exists` in `filesystem.h`

Reply:
Ahh nifty!